.yamato/config/__shared.metafile

```diff
@@ -27,20 +27,123 @@ build_configs:
   - name: mono_apiNet4
     scripting_backend: Mono2x
     api_level: NET_4_6
-color_spaces:
-  - Linear
-  - gamma
+test_platforms:
+  - type: Standalone
+    utr_flags:
+      - [all]: --suite=playmode
+      - [all]: --timeout=1200
+      - [all]: --reruncount=2
+      - [all]: --scripting-backend=<SCRIPTING_BACKEND> 
+      - [all]: --extra-editor-arg="-colorspace=<COLORSPACE>"
+
+      - [Win_DX11, Win_DX12, Win_Vulkan, Win_]: --platform=StandaloneWindows64
+      - [Linux_OpenGlCore, Linux_Vulkan]: --platform=StandaloneLinux64
+      - [OSX_OpenGlCore, OSX_Metal]: --platform=StandaloneOSX
+      - [iPhone_Metal]: --platform=iOS
+      - [Android_OpenGLES3, Android_Vulkan]: --platform=Android
+      
+      - [Win_DX11, Win_DX12, Win_Vulkan, Win_, Linux_OpenGlCore, Linux_Vulkan]: --player-load-path=../../players 
+      - [Android_OpenGLES3, Android_Vulkan, iPhone_Metal]: --player-load-path=players 
+      
+      - [all]: --artifacts_path=test-results
```

Comment:
Would it make sense to group the `[all]` tags at the top of each flags list? I feel it would be easier to visually parse, but that's a personal take on it so feel free to keep it that way if this is intended.

Reply:
I've now grouped them by flag if there is some [platform_api] overriding the [all], to state it clear when a general flag gets overriden, but good point, now that flag order is not affected we can reorder this as we want later!